collections/function/check-params-object-type-of-function.md

```diff
@@ -0,0 +1,18 @@
+~~~ javascript
+const Type = object => Object.prototype.toString.call(object).match(/\[object (.*)\]/)[1]
```

Comment:
Can you change the function name to `getTypeOf`?

Reply:
sure